planning_interface/common_planning_interface_objects/include/moveit/common_planning_interface_objects/common_objects.h

```diff
@@ -49,7 +49,16 @@ boost::shared_ptr<tf::Transformer> getSharedTF();
 
 robot_model::RobotModelConstPtr getSharedRobotModel(const std::string &robot_description);
 
-planning_scene_monitor::CurrentStateMonitorPtr getSharedStateMonitor(const robot_model::RobotModelConstPtr &kmodel, const boost::shared_ptr<tf::Transformer> &tf);
+/**
+ * @brief getSharedStateMonitor
+ *
+ * @param kmodel
+ * @param tf
+ * @param nh An optionnal node handle to pass options
+ * @return
+ */
+planning_scene_monitor::CurrentStateMonitorPtr getSharedStateMonitor(const robot_model::RobotModelConstPtr &kmodel, const boost::shared_ptr<tf::Transformer> &tf,
+    ros::NodeHandle nh = ros::NodeHandle() );
```

Comment:
This is the same story as the CurrentStateMonitor constructor.
Please keep the old function signature around to keep ABI.